src/main/java/frc/robot/Robot.java

```diff
@@ -67,6 +72,9 @@ private void configureBindings() {
     controller.square().whileTrue(swerve.pathFindToLocation(HM_LEFT));
     controller.triangle().whileTrue(swerve.pathFindToLocation(HM_CENTER));
     controller.circle().whileTrue(swerve.pathFindToLocation(HM_RIGHT));
+
+    shooter.setDefaultCommand(shooter.ManualShooterCommand().alongWith(shooter.StartLinearMotor(controllerOperator::getLeftY)));
+    shooter.setDefaultCommand(shooter.ManualShooterCommand().alongWith(shooter.SpeakerShotWithControl(controllerOperator::getLeftX)));
```

Comment:
You can't set two default commands for one subsystem.. choose which one do you want..

(Putting the manualCommand along with the shootWithControlCommand doesnt make any sense)